EXTENSION_POLICY.md

```diff
@@ -92,7 +92,7 @@ The `status` is one of:
 The extension status may be adjusted by the extension [CODEOWNERS](./CODEOWNERS) and/or Envoy
 maintainers based on an assessment of the above criteria. Note that the status of the extension
 reflects the implementation status. It is orthogonal to the API stability, for example, an extension
-with configuration `envoy.foo.v3alpha.Bar` might have a `stable` implementation and
+API marked with `(xds.annotations.v3.file_status).work_in_progress` might have a `stable` implementation and
 `envoy.foo.v3.Baz` can have a `wip` implementation.
```

Comment:
Suggest changing this to say "and an extension with a stable config proto can have a `wip` implementation".